<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Organizator</string>
  </property>
  <widget class="QCalendarWidget" name="cw_kalendarz">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>391</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>160</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pb_kontakty">
      <property name="text">
       <string>Kontakty</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_dodaj">
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_edytuj">
      <property name="text">
       <string>Edytuj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_podglad">
      <property name="text">
       <string>Podgląd</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_usun">
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="lw_spotkania">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>261</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
